src/content/en/tools/chrome-devtools/javascript/get-started.md

```diff
@@ -0,0 +1,274 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Get started with debugging JavaScript using Chrome DevTools in this interactive tutorial.
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+<style>
+.devtools-inline {
+  max-height: 1em;
+  vertical-align: middle;
+}
+</style>
+
+# Get Started with Debugging JavaScript in Chrome DevTools {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+This step-by-step, interactive tutorial teaches you the basic
+workflow for debugging JavaScript in Chrome DevTools. The tutorial shows you
+how to debug one specific issue, but the general workflow you learn is helpful
+for debugging all types of JavaScript bugs.
+
+If you're using `console.log()` to find and fix bugs in your code, consider
```

Comment:
Not sure about this second paragraph-- the tutorial later explains why you might want to use different alternatives to console log, so maybe it isn't necessary?